sdks/python/apache_beam/typehints/trivial_inference_test.py

```diff
@@ -34,20 +34,32 @@ class TrivialInferenceTest(unittest.TestCase):
   def assertReturnType(self, expected, f, inputs=()):
     self.assertEquals(expected, trivial_inference.infer_return_type(f, inputs))
 
+  @unittest.skipIf(sys.version_info >= (3, 6, 0) and
```

Comment:
It may be easier to move the skip declaration to skip entire `class TrivialInferenceTest`.